docs/source/markdown/podman-events.1.md

```diff
@@ -15,6 +15,8 @@ value to `file`.  Only `file` and `journald` are accepted. A `none` logger is al
 available but this logging mechanism completely disables events; nothing will be reported by
 `podman events`.
 
+By default, Podman will stream new events.  Previous events can be listed via `--since` and `--until`.
```

Comment:
Recommend clarifying a bit - `By default, streaming mode is used, printing new events as they occur. Previous events...`